src/BioFSharp.BioTools/Blast.fs

```diff
@@ -167,18 +167,23 @@ module Blast =
 
 
     type BlastParams =
-        | SearchDB of string
-        | Query    of string
-        | Output   of string
-        | OutputType of OutputType
+        | SearchDB     of string
+        | Query        of string
+        | Output       of string
+        | OutputPSSM   of string
+        | OutputType   of OutputType
         | OutputTypeCustom of OutputType * seq<OutputCustom>
-        | Num_threads of int
-        | Max_Hits of int
+        | Num_threads  of int
+        | Max_Hits     of int
+        | Num_Iterations of int
+        | Comp_Based_Stats of int
+        | Word_Size    of int
 
```

Comment:
Are the PsiBlast parameters applicable for normal blast runs or specific for psi blast? If the latter is the case, a seperated type for these commands should be used.

Reply:
@MikhayN , any updates on this?